openedx/core/djangoapps/api_admin/management/commands/create_api_access_request.py

```diff
@@ -0,0 +1,71 @@
+""" Management command to create an ApiAccessRequest for given users """
+import logging
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.auth.models import User
+from django.contrib.sites.models import Site
+
+from openedx.core.djangoapps.api_admin.models import (
+    ApiAccessConfig,
+    ApiAccessRequest,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Create an ApiAccessRequest for the given user
+
+    Example usage:
+        $ ./manage.py lms create_api_request <username> --create-config
+    """
+
+    help = 'Create an ApiAccessRequest for the given user'
+    WEBSITE = 'http://www.test-edx-example-website.edu/'
+    DUMMY_SITE_NAME = 'ApiAccessRequest Management Command Dummy Site'
+    REASON_TEXT = 'Generated by management job create_api_request'
+
+    def add_arguments(self, parser):
+        parser.add_argument('username', type=str)
+        parser.add_argument(
+            '--create-config',
+            action='store_true',
+            help='Create ApiAccessConfig if it does not exist'
+        )
+
+    def handle(self, *args, **options):
+        user = self.get_user(options.get('username'))
+        self.create_api_access_request(user)
+        if options.get('create_config'):
+            self.create_api_access_config()
+
+    def get_user(self, username):
+        try:
+            return User.objects.get(username=username)
+        except User.DoesNotExist:
+            raise CommandError(u'User {} not found'.format(username))
+
+    def create_api_access_request(self, user):
+        try:
+            dummy_site, _ = Site.objects.get_or_create(domain=self.WEBSITE, name=self.DUMMY_SITE_NAME)
+            ApiAccessRequest.objects.create(
+                user=user,
+                website=self.WEBSITE,
+                reason=self.REASON_TEXT,
+                site=dummy_site,
+            )
+            logger.info(u'Created ApiAccessRequest for user {}'.format(user.username))
+        except Exception as e:
+            msg = u'Unable to create ApiAccessRequest for {}. Exception is {}'.format(user.username, e)
+            raise CommandError(msg)
+
+    def create_api_access_config(self):
+        try:
+            _, created = ApiAccessConfig.objects.get_or_create(enabled=True)
```

Comment:
This might get weird because `ApiAccessConfig` is one of those tricky models like @Zacharis278 was dealing with... not sure.